lit/docs/guides/git.lit

```diff
@@ -139,3 +139,203 @@
     }}
   }
 }
+
+\section{
+  \title{Multi-Branch Workflows}{multi-branch-workflows}
+  \omit-children-from-table-of-contents
+
+  Teams may make use of multiple branches for their development. For instance,
+  some teams create feature branches while working on new functionality - once
+  this functionality is ready, the branch will be merged into the main branch
+  and the feature branch will be deleted.
+
+  While a feature is under development, you'll often want to run tests against
+  the feature branch and possibly deploy to a staging environment. To model this
+  in Concourse, you'll need to have a pipeline for each active feature branch.
+  Manually setting (and eventually archiving) a pipeline for each feature branch
+  would be quite a burden. For this type of workflow, Concourse has a few
+  important tools to help you out: the \reference{set-pipeline-step},
+  \reference{schema.step.across}, and \reference{instanced-pipelines}{instanced
+  pipelines}.
+
+  \warn{
+    \reference{schema.step.across} and \reference{instanced-pipelines}{instanced pipelines}
+    are both experimental features, and must be enabled with the feature flags
+    \code{CONCOURSE_ENABLE_ACROSS_STEP} and \code{CONCOURSE_ENABLE_PIPELINE_INSTANCES}, respectively.
+  }
+
+  In this guide, we'll cover:
+
+  \ordered-list{
+    Writing a pipeline to \reference{multi-branch-test-build-deploy} a branch
+    to a staging environment. We'll use \link{Terraform}{https://www.terraform.io/}
+    for our deployment
+  }{
+    \reference{multi-branch-tracking-branches} in a repository; for each
+    branch, we'll set a pipeline (using the \reference{set-pipeline-step} and
+    \reference{schema.step.across})
+  }{
+    Automatically \reference{multi-branch-cleaning-up} after branches get
+    merged or deleted
+  }
+
+  \section{
+    \title{Test, Build & Deploy}{multi-branch-test-build-deploy}
+
+    We'll start out by defining the pipeline that should run for each active
+    branch. For this example, we'll be working with the following \link{sample Go
+    application}{https://github.com/concourse/examples/tree/master/apps/golang}.
+
+    Our pipeline will have three stages:
+
+    \ordered-list{
+      Run unit tests
+    }{
+      Build and upload a binary to a blobstore (in our case, we'll use
+      \link{Google Cloud Storage}{https://cloud.google.com/storage})
+    }{
+      Trigger a \code{terraform apply} to deploy our app to a staging
+      environment. The \link{Terraform module}{https://github.com/aoldershaw/examples/blob/master/terraform/staging/main.tf}
```

Comment:
Reminder to self - a bunch of these links are going to have to change when https://github.com/concourse/examples/pull/4 gets merged

Reply:
Update the links and then this is good to go 👍